tensorboard/main.py

```diff
@@ -31,6 +31,11 @@
 from tensorboard import program
 
 
+def run():
+  """Initializes flags and calls main()."""
+  tf.app.run(main)
```

Comment:
Should we change the `if __name__ == '__main__'` below to just call run()?  Slightly less duplication.

Reply:
Done